apps/web/src/components/marketing/monitor/globe.tsx

```diff
@@ -2,11 +2,15 @@
 
 import { useEffect, useRef } from "react";
 import createGlobe from "cobe";
+import { useMediaQuery } from 'react-responsive';
```

Comment:
Let's try to avoid adding a new package specifically for that.

How about adding the `useMediaQuery` as a separate hook into the folder `@/hooks/use-media-query.ts`:


https://usehooks-ts.com/react-hook/use-media-query

```ts
import { useEffect, useState } from 'react'

export function useMediaQuery(query: string): boolean {
  const getMatches = (query: string): boolean => {
    // Prevents SSR issues
    if (typeof window !== 'undefined') {
      return window.matchMedia(query).matches
    }
    return false
  }

  const [matches, setMatches] = useState<boolean>(getMatches(query))

  function handleChange() {
    setMatches(getMatches(query))
  }

  useEffect(() => {
    const matchMedia = window.matchMedia(query)

    // Triggered at the first client-side load and if query changes
    handleChange()

    // Listen matchMedia
    if (matchMedia.addListener) {
      matchMedia.addListener(handleChange)
    } else {
      matchMedia.addEventListener('change', handleChange)
    }

    return () => {
      if (matchMedia.removeListener) {
        matchMedia.removeListener(handleChange)
      } else {
        matchMedia.removeEventListener('change', handleChange)
      }
    }
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, [query])

  return matches
}
```

Reply:
ok sure